client/src/__tests__/Overview.test.js

```diff
@@ -31,65 +39,151 @@ const mockProduct = {
   ],
 };
 
-const mockStyle = {
-  product_id: '40346',
-  results: [
-    {
-      style_id: 240510,
-      name: 'Black',
-      original_price: '40.00',
-      sale_price: null,
-      'default?': true,
-      photos: [{
-        thumbnail_url: 'https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
-        url: 'https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80'
-      }],
-    },
-  ],
-};
+// const mockStyle = {
+//   product_id: '40346',
+//   results: [
+//     {
+//       style_id: 240510,
+//       name: 'Black',
+//       original_price: '40.00',
+//       sale_price: null,
+//       'default?': true,
+//       photos: [{
+//         thumbnail_url: 'https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+//         url: 'https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80'
+//       }],
+//     },
+//   ],
+// };
 
-const mockReview = {
-    "product": "40346",
-    "page": 0,
-    "count": 5,
-    "results": [
-        {
-            "review_id": 1279314,
-            "rating": 2,
-            "summary": "summary",
-            "recommend": true,
-            "response": null,
-            "body": "Aidan and Nam were here",
-            "date": "2023-03-27T00:00:00.000Z",
-            "reviewer_name": "Andrew",
-            "helpfulness": 4,
-            "photos": []
-        },
-      ]
-}
+describe('Overview Page', () => {
```

Comment:
Overall, great tests for the Overview to check if each component is rendering. I would suggest testing negatives as well, such as if no data is given to see if components don't render